float32 shoulder
float32 upper_arm
float32 forearm
float32 wrist_1
float32 wrist_2
float32 wrist_3
float32 gripper
int64 time